helpers/metamask.ts

```diff
@@ -0,0 +1,20 @@
+export const identifierNeworkLabel = (chainId: number) => {
```

Comment:
use an enumerator, like the one that already exist on the application to map networks

Reply:
Finish